twine-upload.sh

```diff
@@ -130,6 +158,15 @@ if [[ ${INPUT_VERBOSE,,} != "false" ]] ; then
     TWINE_EXTRA_ARGS="--verbose $TWINE_EXTRA_ARGS"
 fi
 
+if [[ ${INPUT_ATTESTATIONS,,} != "false" ]] ; then
+    # NOTE: Intentionally placed after `twine check`, to prevent attestation
+    # generation on distributions with invalid metadata.
+    echo "::debug::Generating and uploading PEP 740 attestations"
```

Comment:
should this really be debug and not normal log level?

Reply:
Normal log level makes sense to me -- in that case surfacing the PEP number probably isn't super helpful, so I'll tweak the message as well.